src/test/java/core/basesyntax/ReturnHandlerTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.ValidationDataException;
+import core.basesyntax.services.actions.ActionHandler;
+import core.basesyntax.services.actions.BalanceActionHandler;
+import core.basesyntax.services.actions.ReturnActionHandler;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class ReturnHandlerTest {
+    private static Storage fruitDB;
+    private static ActionHandler actionHandler;
+
+    @BeforeAll
+    static void createHandlerService() {
+        fruitDB = new Storage();
+        actionHandler = new ReturnActionHandler(fruitDB);
+    }
+
+    @BeforeEach
+    void cleanStorage() {
+        fruitDB.getStorageFruits().clear();
+    }
+
+    @Test
+    void returnHandle_nullStorage_notOk() {
+        assertThrows(ValidationDataException.class,
+                () -> new BalanceActionHandler(null)
+                        .executeAction("banana", 20));
+    }
+
+    @Test
+    void returnHandle_emptyName_ok() {
+        fruitDB.add("banana", 30);
+        assertThrows(ValidationDataException.class,
+                () -> actionHandler
+                        .executeAction("", 20));
+    }
+
+    @Test
+    void returnHandle_nullValue_ok() {
+        fruitDB.add("banana", 30);
+        assertThrows(ValidationDataException.class,
+                () -> actionHandler
+                        .executeAction("banana", null));
+    }
+
+    @Test
+    void returnHandle_negativeValue_ok() {

```

Comment:
negative value - ok? 
ok - means good/success result 
but your result throws an exception.
fix everywhere 
```suggestion
    void returnHandle_negativeValue_notOk() {
```